build.xml

```diff
@@ -109,7 +109,7 @@
    <arg value="--report-checkstyle=${basedir}/build/logs/checkstyle.xml" />
    <arg value="--report-summary" />
    <arg value="--standard=PSR2" />
-   <arg value="--ignore=*.min.*,*/vendor/*,*/PublicAPI/rtfm/*" />
+   <arg value="--ignore=*.min.*,*/vendor/*,*/PublicAPI/rtfm/*,*/Public/js/*" />
    <arg path="${basedir}/src" />
    <arg value="-p" />
   </exec>
```

Comment:
Jenkins currently does all the linting, Travis does the tests? I guess we could keep it like that. I think it might be nice to consolidate into one platform, but separation may make sense. It would be nice if the Jenkins reports were more easily accessible though